deployment/refinery-modules/refinery/manifests/apache2.pp

```diff
@@ -1,9 +1,9 @@
 class refinery::apache2 {
 
-  # must be set to the ELB Idle Timeout value provided in the CloudFormation template
-  # for chunked uploads of large files (MD5 calculation on the server can take a long time)
-  # to be adjusted to a more reasonable value after enabling file uploads to S3
-  $timeout = 1800
+  # must be set to the ELB IdleTimeout value in the CloudFormation template
+  # this is higher than the default of 60 to accommodate long synchronous
+  # processing of metadata during data set import
+  $timeout = 180  # seconds
```

Comment:
How did you pick this number? If the time increases with larger files, how big a file will typically work in 180s?

(If you're tweaking the comment, could you capitalize the first letter and add a period at the end?)

Reply:
180 seconds was more than enough to import each of the several randomly selected ISA-archives. In any case, HTTP requests should not take this long, so if there is more time required to process metadata then this is a larger problem that should be addressed in the metadata processing code.

[Puppet style guide](https://puppet.com/docs/puppet/5.4/style_guide.html#comments) does not address capitalization or punctuation, so I think this is acceptable.

Reply:
I thought we bumped this value initially to handle large file uploads?

Reply:
Large file uploads are sent directly to S3.